examples/post_training_quantization/onnx/mobilenet_v2/requirements.txt

```diff
@@ -0,0 +1,6 @@
+onnx
```

Comment:
onnx and onnxruntime-openvino are already included in requirments.txt of ONNX backend. Should we expose them here?

Reply:
I think, the example has to work if the user installed nncf one of the following ways:
```
pip install nncf
```
```
pip install nncf[onnx]
```
Thus I added onnx and onnxruntime-openvino as dependencies.

Also, I fixed versions of the onnx and onnxruntime-openvino taking them from setup.py.